class.jetpack.php

```diff
@@ -31,7 +37,6 @@
 use \Automattic\Jetpack\Assets\Logo as Jetpack_Logo;
 
 require_once( JETPACK__PLUGIN_DIR . '_inc/lib/class.media.php' );
-require_once( dirname( __FILE__ ) . '/_inc/lib/tracks/client.php' );
 
 class Jetpack {
```

Comment:
I wonder if we should make the `Jetpack` class `final`. Doesn't seem like we'll want to extend it. I remember Woo [does the same](https://github.com/woocommerce/woocommerce/blob/f6d9faa0623d1085eacc94dee2ad75750d916ed4/includes/class-woocommerce.php#L16).